src/Pages/MenuList/MenuList.js

```diff
@@ -1,9 +1,126 @@
 import React from 'react';
 import './MenuList.scss';
+import { DAYS } from './MenuListDays';
+import { FILTER } from './MenuListFilter';
+import List from './List';
+import { clearConfigCache } from 'prettier';
 
 class MenuList extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      tab: '배송일별',
+      cate: '전체',
+      snackList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/MenuList/menuData.json', {
+      method: 'GET',
+    })
```

Comment:
```suggestion
    fetch('/data/MenuList/menuData.json', {
```
이렇게 줄여볼 수 있습니다. get method가 기본이라 없어도 될 것 같습니다.